src/search_engines.js

```diff
@@ -444,6 +444,79 @@ class GoogleSearch {
     }
     return false;
   }
+
+  changeImageSize(size) {
+    const openTool = document.querySelector(
+        '[class="PNyWAd ZXJQ7c"][jsname="I4bIT"]');
```

Comment:
The class names and other attributes seem brittle (it's highly likely that Google will change them).
Did you try to find something that is less likely to break?
Maybe there isn't a good alternative, but I wanted to verify.

Reply:
Hi, I tried to find an alternative but can't find one, sorry about that.